polyfill/lib/timezone.mjs

```diff
@@ -222,6 +222,9 @@ export class TimeZone {
     if (!ES.IsTemporalTimeZone(this)) throw new TypeError('invalid receiver');
     return this.name;
   }
+  toJSON() {
+    return this.toString();
```

Comment:
why would this observably look up `toString` as opposed to just using the same algorithm?

Reply:
This is very much against the design of JS imo, and I don't think it's a good change.

Reply:
TimeZones can be subclassed, or provided as plain objects with function properties, and need only implement `getPossibleAbsolutesFor()`, `getDateTimeFor()`, and `toString()`. The other methods are defined in terms of these three.

Reply:
Why is that a desirable property? I think a subclass should have to override `toJSON`, just like Arrays and Objects.

Reply:
The subclassable API was designed such that you'd only have to implement the essential parts of a time zone (one-to-many datetime-to-absolute, one-to-one absolute-to-datetime, ID string, and optionally offset transitions) and could chain up or call `Temporal.TimeZone.prototype` methods for everything else. I think that's desirable.

But, since we are now in the stage of looking for feedback on the API, please open a new issue and tag it with "feedback"! :smile: